slices/ca-certificates-java.yaml

```diff
@@ -0,0 +1,19 @@
+package: ca-certificates-java
+
+essential:
+  - ca-certificates-java_copyright
+
+slices:
+  data:
+    essential:
+      - ca-certificates_data
+      - libnss3_libs
+    contents:
+      /etc/default/cacerts:
+      /etc/ssl/certs/java/:
+      # we need to run `keytool` in postinst, but we can't yet do
+      # this from a mutation script
```

Comment:
what operations are missing in Starlark?

Reply:
I think that the key blocker - we need to run java command from ca-certificates-java so this is a chicken and egg problem.

Reply:
I have dropped ca-certificates-java dependency as the package only installs jar file that contains a maintainer script implementation.